### DISCUSSION TITLE
Shouldn't Fief Limit the Workspace Creation to an Admin?

### DISCUSSION THREAD

**Discussion by ScottFred** [fief-dev/fief#313] [Q&A]
Related to [this question](https://github.com/orgs/fief-dev/discussions/198), it seems that ANY user is able to create a new Fief Workspace.  I would think that the ONLY person to create a Fief Workspace should be the admin user.  (Maybe this is a self-hosted problem only?  Admittedly, I haven't tried it yet on my Fief Cloud instance.)

**Comment by frankie567** [comment]
Well, technically, yes. It's a side-effect of the project being designed at first for cloud usage (on Fief Cloud, anyone can create a workspace, of course).

I admit it's not ideal, and that we should probably add a setting about it.
